tests/drivers/dma/loop_transfer/boards/nucleo_wb55rg.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_DMA_LOOP_TRANSFER_DRV_NAME="DMAMUX_1"
+CONFIG_DMA_LOOP_TRANSFER_CHANNEL_NR=11
+CONFIG_DMA_MCUX_TEST_SLOT_START=0
```

Comment:
This is a MCUX ("depends on DMA_MCUX_EDMA and depends on (SOC_SERIES_KINETIS_K6X || SOC_SERIES_KINETIS_KE1XF)") specific property.  We should not be able to use it for STM32.

If requested to pass the test, then the test should be reworked.

Reply:
removed